From 369b1791e3fff5ea16cdf1fe9ba5c374af7faa43 Mon Sep 17 00:00:00 2001
From: Jakub Filak <jfilak@redhat.com>
Date: Tue, 29 Apr 2014 13:24:29 +0200
Subject: [LIBREPORT PATCH 10/10] reporter-upload: more descriptive message
 about missing URL

Only one line is possible. Hopefully, this message is better than the
previous one.

Related to #259

Signed-off-by: Jakub Filak <jfilak@redhat.com>
---
 src/plugins/reporter-upload.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/plugins/reporter-upload.c b/src/plugins/reporter-upload.c
index b903aa1..1ef38e7 100644
--- a/src/plugins/reporter-upload.c
+++ b/src/plugins/reporter-upload.c
@@ -58,7 +58,7 @@ static int create_and_upload_archive(
     const char* opt = getenv("Upload_URL");
     if (!opt)
         opt = get_map_string_item_or_empty(settings, "URL");
-    char *url = opt[0] != '\0' ? xstrdup(opt) : ask_url(_("Upload URL is not provided by configuration. Please enter upload URL:"));
+    char *url = opt[0] != '\0' ? xstrdup(opt) : ask_url(_("Please enter a URL (scp, ftp, etc.) where the problem data is to be exported:"));
 
     /* Create a child gzip which will compress the data */
     /* SELinux guys are not happy with /tmp, using /var/run/abrt */
-- 
1.8.3.1

